packages/core/src/Stores/Helpers/chart-markers.js

```diff
@@ -68,13 +68,19 @@ const addLabelAlignment = (tick, idx, arr) => {
 
 const createTickMarkers = contract_info => {
     const is_accumulator = isAccumulatorContract(contract_info.contract_type);
-    const is_contract_closed = contract_info.status && contract_info.exit_tick_time;
+    const is_contract_closed = contract_info.status !== 'open';
     const available_ticks = (is_accumulator && contract_info.audit_details?.all_ticks) || contract_info.tick_stream;
     const tick_stream = unique(available_ticks, 'epoch').map(addLabelAlignment);
     const result = [];
 
     if (is_contract_closed && is_accumulator) {
-        tick_stream.length = tick_stream.findIndex(tick => tick.epoch === contract_info.exit_tick_time) + 1;
+        const { exit_tick_time, tick_stream: ticks } = contract_info || {};
+        if (exit_tick_time && tick_stream.every(({ epoch }) => epoch !== exit_tick_time)) {
+            // sometimes exit_tick is present in tick_stream but missing from audit_details
+            tick_stream.push(ticks[ticks.length - 1]);
+        }
+        const length = tick_stream.findIndex(({ epoch }) => epoch === exit_tick_time) + 1;
```

Comment:
```suggestion
        const exit_tick_count = tick_stream.findIndex(({ epoch }) => epoch === exit_tick_time) + 1;
```
to make it more understandable (at least for me 😆 )